### PR TITLE
mail-server: Portfile and launch daemon improvements and bugfixes

### PR DESCRIPTION
**Pull request by essandess**
mail-server: Portfile and launch daemon improvements and bugfixes

* Ensure that launch daemons run at load (necessary)
* Apply review comments from PR https://github.com/macports/macports-ports/pull/4978

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G1012
Xcode 11.2 11B52

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: mail/mail-server/Portfile [hunk 1]
```diff
@@ -47,9 +47,19 @@ depends_lib-append      port:apache-solr8 \
 
 depends_run-append      port:clamav-server
 
-variant initialize \
-    description {Initialize all configuration files. Existing
-        configurations files are not overwritten by default.} {}
+variant initializealways \
```
Comment (mf2k) [inline mail/mail-server/Portfile]: Other variants with multiple words use an underscore. So change this to `initialize_always`

Comment (essandess) [reply mail/mail-server/Portfile]: Thanks, done.

Comment (essandess) [reply mail/mail-server/Portfile]: And also with https://github.com/macports/macports-ports/pull/4978 and https://github.com/macports/macports-ports/pull/5643.

**Comment by mf2k** [comment]
I'm not understanding why it is failing on the buildbot.

**Comment by essandess** [comment]
Me either. I see this on several of my previously successfully-built PRs that I pushed minor commits to in the past ~18 hours. I’m blaming azure for now.

**Pull request merged**